scheduler/scheduler.go

```diff
@@ -172,9 +176,32 @@ func NewEtcdScheduler(
 		memPerTask:                   memPerTask,
 		offerRefuseSeconds:           offerRefuseSeconds,
 		reconciliationInfo:           map[string]string{},
+		constraints:                  Constraints{},
 	}
 }
 
+// AddRawConstraints -- parses a constraint as they'd be added by Marathon.
+func (s *EtcdScheduler) AddRawConstraints(statements string) error {
+	for _, tuple := range strings.Split(statements, ";") {
+		statement := strings.SplitN(tuple, ":", 3)
+		switch op := strings.ToUpper(statement[1]); op {
```

Comment:
- Constraints marathon constraints LIKE, UNLIKE. 
- UNIQUE(enforced by existing framework switch), MAX_PER, CLUSTER, GROUP_BY are explicitly ignored 
- The entire marathon constraint group may be passed wholesale but only LIKE,UNLIKE are interpreted
- All constraints must pass just like marathon i.e assume AND not OR between constraints
- Just like Marathon, the form is:
    - noun:verb:optional*;
- See [go-related commit](https://github.com/adobe-platform/etcd-mesos/pull/1/commits/2507722c5b8f45e50405231758d75e8852a001ff) for golang constraint processing

- Everything else involves pulling out GoDeps/*
- git submodules
- Using go 1.8.1
- 2-phase build like FD, capcom, go-metronome, skopos, skms-gate etc
- Makefile to be  like FD, capcom, go-metronome, skopos, skms-gate etc